packages/connectivity/src/scp-cf/identity-service.ts

```diff
@@ -4,12 +4,12 @@ import type { DestinationOptions } from './destination';
 
 /**
  * @internal
- * Make a token exchange from IAS token to XSUAA token.
- * @param options - Configuration for how to retrieve destinations from the destination service.
- * @returns Exchanged token.
+ * Make a token exchange from IAS token to XSUAA token using credentials from destination service.
+ * @param jwt - The IAS JWT to exchange.
+ * @returns Exchanged XSUAA token.
  */
-export async function exchangeToken(jwt: string): Promise<string> {
-  return jwtBearerToken(jwt, 'xsuaa');
+export async function exchangeTokenToXsuaaToken(jwt: string): Promise<string> {
```

Comment:
[req] This function is located in `scp-cf` folder and was meant to be more generic to exchange IAS token to the bound XSUAA service token. Since now its purpose changed to only retrieve token for destination service, I would suggest to either keep the original one or move this to subfolder `scp-cf/destination`.

Reply:
We discussed using `jwtBearerToken` directly instead of the wrapper function.